src/scene/sprite-animated/AnimatedSprite.ts

```diff
@@ -110,19 +127,39 @@ export class AnimatedSprite extends Sprite
 
     /** The texture index that was displayed last time. */
     private _previousFrame: number;
-
     /**
-     * @param textures - An array of {@link Texture} or frame
-     *  objects that make up the animation.
-     * @param {boolean} [autoUpdate=true] - Whether to use Ticker.shared to auto update animation time.
+     * @param frames - Collection of textures or frames to use.
+     * @param autoUpdate - Whether to use Ticker.shared to auto update animation time.
+     */
+    constructor(frames: AnimatedSpriteFrames, autoUpdate?: boolean);
```

Comment:
Should we also mark this old constructor as `@deprecated`?

Reply:
Also, I'd rather be conservative with the deprecation to get it in a patch release.